bc_obps/registration/email/emailService.py

```
@@ -0,0 +1,81 @@
+            else:
+                raise Exception("Failed to retrieve CHES access token")
+
+    def _make_request(self, endpoint, method='GET', data: any = None):
+        headers = {"Authorization": f'Bearer {self.token}'} if self.token else {}
+        print(f'Received request to {method} to {endpoint} with payload {data}')
+        if method == 'GET':
+            response = requests.get(self.apiUrl + endpoint, headers=headers, timeout=10)
+        elif method == 'POST':
+            response = requests.post(self.apiUrl + endpoint, headers=headers, json=data, timeout=10)
+        else:
+            raise ValueError("Invalid HTTP method")
+
+        return response
+
+    def health_check(self):
+        self._get_token()
+        try:
+            response = self._make_request("/health")
+            return response.json()
+        except Exception as exc:
+            print("Exception in /email/health_check ", str(exc))
+
+    def get_message_status(self, msgId: UUID):
+        self._get_token()
+        try:
+            response = self._make_request(f'/status/{msgId}')
+            return response.json()
+        except Exception as exc:
+            print(f'Exception retrieving message status for {msgId} - ', str(exc))
+
+    def send_email(self, email_data: dict):
+        self._get_token()
+        try:
+            response = self._make_request(
+                '/email',
+                method='POST',
+                data=email_data,
+            )
+            return response.json()
+        except Exception as exc:
+            print("Exception in send_email ", str(exc))
+
+    def merge_template_and_send(self, email_template_data: dict):
+        self._get_token()
+        try:
+            response = self._make_request("/emailMerge", method='POST', data=email_template_data)
+            return response.json()
+        except Exception as exc:
+            print('Exception in merging template and sending! ', str(exc))
```

Could we raise an exception instead of printing the message to the console?